### PR TITLE
[docs] Fixes issue #72389 

### PR DESCRIPTION
**Pull request by JaroslavKlech**
##### SUMMARY
Adds a suggested note mentioned in https://github.com/ansible/ansible/issues/72389#issuecomment-1234750451 and polishes the `sample_setup.rst` file in general.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME
docs.ansible.com

##### ADDITIONAL INFORMATION

https://github.com/ansible/ansible/issues/72389

### CODE REVIEW

File: docs/docsite/rst/tips_tricks/sample_setup.rst
Comment (tremble) [inline docs/docsite/rst/tips_tricks/sample_setup.rst]:
```suggestion
You have learned about playbooks, inventory, roles, and variables. This section combines all those elements, and outlines a sample setup for automating a web service. You can find more example playbooks that illustrate these patterns in our `ansible-examples repository <https://github.com/ansible/ansible-examples>`_. (NOTE: These examples do not use all of the latest features, but are still an excellent reference.).
```

**Review by tremble — CHANGES_REQUESTED** [review]
"These" in the note refers to the examples rather than elements.  I'd leave it to folks more knowledgable than I if the change in voice is desirable.

**Comment by tremble** [comment]
Fixes: https://github.com/ansible/ansible/issues/72389

File: docs/docsite/rst/tips_tricks/sample_setup.rst
Comment (oraNod) [inline docs/docsite/rst/tips_tricks/sample_setup.rst]:
```suggestion
With this setup, you can configure your entire infrastructure by running ``site.yml``. Alternatively, to configure just a portion of your infrastructure, run ``webservers.yml``. This is similar to the Ansible ``--limit`` parameter but a little more explicit:
```

File: docs/docsite/rst/tips_tricks/sample_setup.rst
Comment (oraNod) [inline docs/docsite/rst/tips_tricks/sample_setup.rst]:
```suggestion
The sample setup illustrates a typical configuration topology. When you do multi-tier deployments, you will likely need some additional playbooks that hop between tiers to roll out an application. In this case, you can augment ``site.yml`` with playbooks like ``deploy_exampledotcom.yml``. However, the general concepts still apply. With Ansible you can deploy and configure using the same utility. Therefore, you will probably reuse groups and keep the OS configuration in separate playbooks or roles from the application deployment.
```

File: docs/docsite/rst/tips_tricks/sample_setup.rst
Comment (oraNod) [inline docs/docsite/rst/tips_tricks/sample_setup.rst]:
```suggestion
If a playbook has a :file:`./library` directory relative to its YAML file, you can use this directory to add Ansible modules automatically to the module path. This organizes modules with playbooks. For example, see the directory structure at the start of this section.
```

**Review by oraNod — APPROVED** [review]
@JaroslavKlech Thanks for the changes! I've made some minor suggestions on wording. Feel free to take them, leave them, or modify them. Cheers!

**Comment by JaroslavKlech** [comment]
@oraNod and @tremble, I committed your suggestions. I think they were valid points.

**Comment by oraNod** [comment]
@JaroslavKlech Cheers. I'd like to wait for @samccann to have a quick look before we merge. Should be today though. Thanks again!

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @JaroslavKlech for the detailed Ansible docs fix and Welcome to Ansible!

**Comment by oraNod** [comment]
Thanks @samccann and, yes, welcome @JaroslavKlech ! Great first docs contribution. :tada:

**Pull request merged**
